src/objects/Sprite.js

```diff
@@ -101,9 +101,18 @@ Sprite.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		return function raycast( raycaster, intersects ) {
 
 			worldScale.setFromMatrixScale( this.matrixWorld );
-			viewWorldMatrix.getInverse( this.modelViewMatrix ).premultiply( this.matrixWorld );
+
+			viewWorldMatrix.copy( raycaster.camera.matrixWorld );
```

Comment:
Need to check if the ```raycaster.camera``` exists?

Reply:
It assumes `Raycaster.setFromCamera()` was called previously. This PR is just a proof-of-concept.